core/src/main/java/org/apache/accumulo/core/metadata/TabletFileUtil.java

```diff
@@ -0,0 +1,40 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.apache.accumulo.core.metadata;
+
+import org.apache.hadoop.fs.Path;
+
+/**
+ * Utility class for validation of metadata tablet files.
+ */
+public class TabletFileUtil {
+
+  /**
+   * Validate if string is a valid path. Return same string or throw exception if not valid. This
+   * was added to facilitate more use of TabletFile over String but this puts the validation in one
+   * location in the case where TabletFile can't be used.
+   */
+  public static String validate(String path) {
+    Path p = new Path(path);
+    if (p.toUri().getScheme() == null) {
+      throw new IllegalArgumentException("Invalid path provided, no scheme in " + path);
+    }
+    return path;
```

Comment:
Did we want to have this normalize the path also by returning the normalized version from `Path`?

Reply:
I think that would be good.  You thinking return the ```Path``` object or ```p.toString()```?

Reply:
Not sure. Probably Path, but it depends on caller needs. I haven't investigated.

Reply:
I think these could also get replaced with TabletFile or Ample as well.  So I will just leave as String.  If we can replace them all, this class will go away